package.json

```diff
@@ -20,6 +20,17 @@
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-component:react": "yarn workspace @metamask/design-system-react create-component",
     "create-package": "ts-node scripts/create-package",
+    "figma:connect:react": "figma connect --config packages/design-system-react/figma.config.json",
+    "figma:connect:react-native": "figma connect --config packages/design-system-react-native/figma.config.json",
+    "figma:connect:publish": "yarn figma:connect:publish:react && yarn figma:connect:publish:react-native",
+    "figma:connect:publish:dry-run": "yarn figma:connect:publish:react:dry-run && yarn figma:connect:publish:react-native:dry-run",
+    "figma:connect:publish:react": "figma connect publish --config packages/design-system-react/figma.config.json",
+    "figma:connect:publish:react:dry-run": "figma connect publish --dry-run --config packages/design-system-react/figma.config.json",
+    "figma:connect:publish:react-native": "figma connect publish --config packages/design-system-react-native/figma.config.json",
+    "figma:connect:publish:react-native:dry-run": "figma connect publish --dry-run --config packages/design-system-react-native/figma.config.json",
+    "figma:connect:unpublish": "yarn figma:connect:unpublish:react && yarn figma:connect:unpublish:react-native",
+    "figma:connect:unpublish:react": "figma connect unpublish --config packages/design-system-react/figma.config.json",
+    "figma:connect:unpublish:react-native": "figma connect unpublish --config packages/design-system-react-native/figma.config.json",
```

Comment:
I did consider consolidating these scripts but because we need to add individual config files and components specific to React and React Native I think this individual scripts make sense. Especially for the interactive connect scripts.

```
yarn figma:connect:react
yarn figma:connect:react-native 
```